src/hooks/useToken.ts

```diff
@@ -0,0 +1,20 @@
+import { useSelector } from 'react-redux';
+
+import { getTokenPrices } from '@/constants/assets';
+import { State } from '@/interfaces/redux';
+import { getUsdAvailableAmount } from '@/screens/flows/Send/utils';
+
+export const useToken = (tokenId: string) => {
+  const { assets } = useSelector((state: State) => state.user);
+  const { icpPrice } = useSelector((state: State) => state.icp);
+  const token = assets.find(({ canisterId }) => canisterId === tokenId);
```

Comment:
is there a chance that the token is not found in the array? should we cover that case?